src/gui/search/searchjobwidget.cpp

```diff
@@ -151,6 +151,9 @@ SearchJobWidget::SearchJobWidget(SearchHandler *searchHandler, QWidget *parent)
     connect(searchHandler, &SearchHandler::searchFinished, this, &SearchJobWidget::searchFinished);
     connect(searchHandler, &SearchHandler::searchFailed, this, &SearchJobWidget::searchFailed);
     connect(this, &QObject::destroyed, searchHandler, &QObject::deleteLater);
+
+    m_enterHotkey = new QShortcut(Qt::Key_Return, m_ui->resultsBrowser, nullptr, nullptr, Qt::WidgetShortcut);
```

Comment:
`QShortcut *enterHotkey = new QShortcut ...`
And revert the changes in header file.

Reply:
But that's how it's done in many other places?

Reply:
I would say forget about other places... you're setting `m_ui->resultsBrowser` as enterHotkey's parent, thus there is no reason to keep it in class variable.